Gemfile.template

```diff
@@ -11,7 +11,7 @@ gem "rake", "~> 12"
 gem "ruby-progressbar", "~> 1"
 gem "logstash-output-elasticsearch", ">= 10.4.2"
 gem "childprocess", "~> 4", :group => :build
-gem "fpm", "~> 1.13", :group => :build
+gem "fpm", "~> 1.13.1", :group => :build
```

Comment:
Please add a comment linking to the relevant fpm issue

Reply:
blame should lead us to the PR desc here (includes upstream link) but added a comment as well, thanks.